docs/RNCamera.md

```diff
@@ -500,7 +507,7 @@ The promise will be fulfilled with an object with some of the following properti
 
  - `maxFileSize` (int greater than 0). Specifies the maximum file size, in bytes, of the video to be recorded. For 1mb, for example, use 1\*1024\*1024. If nothing is specified, no size limit will be used.
 
- - `mute` (any value). If this flag is given in the option with any value, the video to be recorded will be mute. If nothing is specified, video will NOT be muted.
+ - `mute` (any value). (*This value will automatically be set to true if the `captureAudio` has not been passed to the Camera component*) If this flag is given in the option with any value, the video to be recorded will be mute. If nothing is specified, video will NOT be muted.  
```

Comment:
Can you please add a note that you should probably use `captureAudio` instead and this API could become deprecated in the future?

Reply:
Sure

Reply:
Should now be fixed.